src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/PackageTreeWriter.java

```diff
@@ -147,8 +147,7 @@ protected void addLinkToMainTree(Content div) {
         Content span = HtmlTree.SPAN(HtmlStyle.packageHierarchyLabel,
                 contents.packageHierarchies);
         div.add(span);
-        HtmlTree ul = new HtmlTree (TagName.UL);
-        ul.setStyle(HtmlStyle.horizontal);
+        HtmlTree ul = HtmlTree.UL(HtmlStyle.horizontal);
         ul.add(getNavLinkMainTree(resources.getText("doclet.All_Packages")));
         div.add(ul);
```

Comment:
This is a general comment that is not specific to this instance.

You could also simplify the code further by using method chaining, even to the point of (in this case) eliminating the need for the local variable.